packages/next-starter/src/modules/shared/mirage/mirage.ts

```diff
@@ -75,6 +75,14 @@ export function makeServer({ environment = 'test' } = {}) {
       for (const route in RoutesRegistration) {
         registerRoute(RoutesRegistration[route]);
       }
+
+       this.passthrough((request) => {
+        if (
+          request.url === "/_next/static/development/_devPagesManifest.json"
+        ) {
+          return true;
+        }
+      });
```

Comment:
How about declaring a const variable at the top of the file, having all the whitelisted passthrough path? 

Actually, I believe, bypassing all path with `/_next`, should be there by default?